controller/adminController.ts

```diff
@@ -0,0 +1,10 @@
+import { Request, Response, Application } from "express"
+import { Login } from "../model/auth"
+const authService = require("../service/AuthService");
+
+module.exports = function(app: Application) {
+
+    app.get('/admin-menu', async (req: Request, res: Response) => {
+        res.render('admin-menu')
+    });
```

Comment:
I would have 1 menu route that loads the correct view depending on the admin logic